resources/public/css/style.css

```diff
@@ -63,6 +63,10 @@ table {
 	border-spacing: 0;
 }
 
+html {
```

Comment:
this style sheet isn't been read by the client (at least in the production uberjar) We believe Larry used this one for testing only. 

Probably needs to be cleaned up. (Along with index.html and blank.html)

Reply:
Do you want me to delete this file then?

Reply:
I've quickly looked into this, I cannot find a reference to `index.html` anywhere and `blank.html` also is no longer in use. there still is a function for `blank.html` in `routes.clj` called `empty-index` but it's not used anywhere. So I think I can safely remove them?

Reply:
I've seen that before and gone wtf.   Just haven't had time to do general project maintenance so thank you for taking a look at it.

Please remove them and test it.  Doesn't look like to me like `empty-index` is used so that as well. 
Probably remove all of that 'lint' since it isn't used, confuses new (and currrent) dev's on the project.

Reply:
and `\resources\.ebextensions` since it is used for AWS deployments.